packages/core/components/DataTable/components/ChartRows.tsx

```diff
@@ -0,0 +1,71 @@
+import LegendCircle from '@cdc/core/components/LegendCircle'
+import { customSort } from '../helpers/customSort'
+import { getSeriesName } from '../helpers/getSeriesName'
+import { DataTableProps } from '../DataTable'
+import { getChartCellValue } from '../helpers/getChartCellValue'
+import { getDataSeriesColumns } from '../helpers/getDataSeriesColumns'
+
+type ChartRowsProps = DataTableProps & {
+  rows: string[]
+  isVertical: boolean
+  sortBy: { colIndex; column }
+}
+
+const ChartRows = ({ rows, runtimeData, config, isVertical, sortBy, colorScale }: ChartRowsProps) => {
+  const dataSeriesColumns = getDataSeriesColumns(config, isVertical, runtimeData)
+
+  const dataSeriesColumnsSorted = () => {
+    if (!sortBy && sortBy.colIndex === null) return dataSeriesColumns
+    return dataSeriesColumns.sort((a, b) => {
+      if (sortBy.column === '__series__') return customSort(a, b, sortBy, config)
+      let row = runtimeData.find(d => d[config.xAxis?.dataKey] === sortBy.column)
+
+      const rowIndex = runtimeData[sortBy.colIndex - 1]
+      if (row) {
+        return customSort(row[a], row[b], sortBy, config)
+      }
+      if (row === undefined && rowIndex) {
+        return customSort(rowIndex[a], rowIndex[b], sortBy, config)
+      }
+    })
+  }
+
+  if (isVertical) {
+    return (
+      <>
+        {rows.map((row, index) => {
+          return (
+            <tr key={`${row}__${index}`} role='row'>
+              {dataSeriesColumns.map((column, j) => {
+                ;<td key={`${row}__${index}_${j}`} tabIndex={0} role='gridcell' id={`${runtimeData[config.runtime?.originalXAxis?.dataKey]}--${row}`}>

```

Comment:
@adamdoe @mpallansch @Atash3000 does anyone know what the id is doing on this td cell? is there some popup or something that's referencing it? Can it be removed?